packages/langium/src/lsp/workspace-symbol-provider.ts

```diff
@@ -76,4 +85,9 @@ export class DefaultWorkspaceSymbolProvider implements WorkspaceSymbolProvider {
             return undefined;
         }
     }
+
+    async resolveSymbol(symbol: WorkspaceSymbol, _cancelToken?: CancellationToken): Promise<WorkspaceSymbol> {
```

Comment:
The interface method is optional, so there's no need to provide a default value here.

Reply:
Removed no-op method implementations.